test/node-fetch/response.js

```diff
@@ -137,13 +136,13 @@ describe('Response', () => {
     })
   })
 
-  it('should support stream as body', () => {
-    const body = stream.Readable.from('a=1')
-    const res = new Response(body)
-    return res.text().then(result => {
-      expect(result).to.equal('a=1')
-    })
-  })
+  // it('should support stream as body', () => {
+  //   const body = stream.Readable.from('a=1')
+  //   const res = new Response(body)
+  //   return res.text().then(result => {
+  //     expect(result).to.equal('a=1')
+  //   })
+  // })
```

Comment:
Shouldn’t it support async iterable as body?

Reply:
That would make sense - fetch spec doesn't explicitly forbid/allow async iterables so it wouldn't be included in a webidl check, but undici does. I'll add support for them back.